src/components/about/Index.jsx

```diff
@@ -3,10 +3,32 @@ import { getTranslate, getActiveLanguage } from 'react-localize-redux';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Connect from '../Connect';
-import Penn from '../../images/penn.png';
-import JTSLogo from '../../images/jts_logo.png';
-import Princeton from '../../images/princeton_geniza.png';
-import Genizah from '../../images/genizah_research.png';
+
+import ImgBodleian from '../../images/partners/bodleian.png';
+import ImgCambridge from '../../images/partners/cambridge.png';
+import ImgElijah from '../../images/partners/Elijah.png';
+import ImgGenizah from '../../images/partners/genizah.png';
+import ImgJTS from '../../images/partners/jts.png';
+import ImgManchester from '../../images/partners/manchester.png';
+import ImgPenn from '../../images/partners/penn.png';
+import ImgPrinceton from '../../images/partners/princeton.png';
+import ImgZooniverse from '../../images/partners/zooniverse-word-black.png';
+
+import ContentIntroEn from './content-intro-en.jsx';
+import ContentIntroHe from './content-intro-he.jsx';
+import ContentIntroAr from './content-intro-ar.jsx';
+
+import ContentGenizaEn from './content-geniza-en.jsx';
+import ContentGenizaHe from './content-geniza-he.jsx';
+import ContentGenizaAr from './content-geniza-ar.jsx';
+
+import ContentResearchEn from './content-research-en.jsx';
+import ContentResearchHe from './content-research-he.jsx';
+import ContentResearchAr from './content-research-ar.jsx';
```

Comment:
Were these imported in such a way, as opposed to inserting them into locales, for styling reasons?

Reply:
I decided to package whole sections into components to make the content easier to read and review (and contrast between different languages), and so the stylings could be managed much more easily. "Easy to read" is my personal opinion though; it's not too difficult for me to change the code to use `/locales/your-language-here.js` if we deem it more suitable for long-term code maintenance.

Reply:
I think we should remove this as it seems awkward to link to a Google document on our page. @snblickhan - do you agree with this? The about section contains the following sentence in English and Hebrew "To learn about the history of the collections the fragments you are transcribing come from, [click here](https://docs.google.com/document/d/1uxY_EgX9qUp-oQMFbFboZvMs3t-nd9FLLibhVO37xdA/edit) or scroll down."